sql/core/src/test/scala/org/apache/spark/status/api/v1/sql/SqlResourceSuite.scala

```diff
@@ -82,6 +82,7 @@ object SqlResourceSuite {
 
     new SQLExecutionUIData(
       executionId = 0,
+      rootExecutionId = 0,
```

Comment:
For testing purpose, let's use a different value from `executionId`

Reply:
done